pkg/operator/api/api.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/dapr/dapr/pkg/logger"
 	operatorv1pb "github.com/dapr/dapr/pkg/proto/operator/v1"
 	"github.com/golang/protobuf/ptypes/empty"
+	pkgerror "github.com/pkg/errors"
```

Comment:
Importing as `pkgerror` to keep this import uniform across files. We seem to use `errors` package as well. Therefore in order to reduce the ambiguity pkgerror is used.

Reply:
Actually, I noticed that pkg errors is mostly a drop-in replacement for the `errors` package, meaning it has the `errors.New` functionality. I think it's better if we do not alias this to `pkgerrors` and replace the `errors` package wherever it appears.

Reply:
@yaron2 are you referring to this? 

https://github.com/pkg/errors/blob/614d223910a179a466c1767a985424175c39b465/errors.go#L102

I just want us to agree on before I make the required changes.

Reply:
Yep.

Reply:
Fixed this. How the error will look like can be seen here. 

https://play.golang.org/p/sliZGohb-7D